### DISCUSSION TITLE
Overclicking overlapping sliders lead to a sliderbreak(miss) in osu lazer?

### DISCUSSION THREAD

**Discussion by ProMedic4U** [ppy/osu#26312] [Q&A] [answered]
I'm a 6-digit player and I sometimes double tap when stream ends with a slider, with an extra click. I found that in osu lazer, this leads to a sliderbreak(of course, a slider miss). Replay files have been attached. [replays.zip](https://github.com/ppy/osu/files/13804921/replays.zip)
Beatmaps: 
https://osu.ppy.sh/beatmapsets/738656#osu/1558875
https://osu.ppy.sh/beatmapsets/63563#osu/187727
Here are screenshots of the patterns I sliderbreak(miss) on.
![screenshot085](https://github.com/ppy/osu/assets/59735928/9f9fbcae-b445-4ff4-b8b4-319e1818a10d)
![screenshot086](https://github.com/ppy/osu/assets/59735928/b6bf661b-a18f-47bc-b283-2a33588d448d)
I suppose if the sliders are overlapped and their time windows are close enough, then overclick always leads to a miss. Obviously, it doesn't work this way in osu stable. So, I want to know whether this is the new mechanic to go or this could be reconsidered. Since many people share this tapping style (Cookiezi always clicks one more time on these patterns to let his non-dominant finger hold the slider at the end), I think many players will be affected.

**Comment by bdach** [comment, marked as answer]
Duplicate of https://github.com/ppy/osu/issues/24626

This should not happen with classic mod active, but yes, by default slider heads will not block input to objects underneath them that have been hit anymore.

**Reply by ProMedic4U** [reply]
Thank you. But it still feels a bit weird as the old mechanic(overlapped objects related) works fine 
and more intuitive to me, personally. The new mechanic is really punishing though it is still kind of "skill issue". I really hope there is a new way to implement new notelock mechanic without affecting gameplay on maps with such patterns.
